Sofa/Component/StateContainer/src/sofa/component/statecontainer/MechanicalObject.inl

```diff
@@ -1032,7 +1032,7 @@ void MechanicalObject<DataTypes>::init()
     //case if X0 has been set but not X
     if (read(core::vec_id::read_access::restPosition)->getValue().size() > x_wA.size())
     {
-        vOp(core::execparams::defaultInstance(), core::VecId::position(), core::VecId::restPosition());
+        vOp(core::execparams::defaultInstance(), sofa::core::vec_id::write_access::position, sofa::core::vec_id::read_access::restPosition);
```

Comment:
One is write and the other one is read, is that correct (seem right to me, we don't want to override the restPosition, but I prefer asking)